src/app/places/places.component.ts

```diff
@@ -2,6 +2,23 @@ import { Component, OnInit } from '@angular/core';
 import { PlacesService } from '../places.service';
 import { Place } from '../../models/place';
 import { PaginatePipe } from 'ngx-pagination';
+import { ActivatedRoute, Router } from '@angular/router';
+/*
+sort: string | undefined;
+// category: string | undefined;
+
+constructor(private route: ActivatedRoute) {}
+
+ngOnInit() {
+  this.route.queryParams
+  // .filter(params => params.category)
+  .subscribe((params) => {
+    console.log(params);
+    this.sort = params.sort;
+    // this.category = params.category;
+    console.log(this.sort);
+  });
+}*/
```

Comment:
I would not include code that is commented out - referring to my previous comment: if it's not needed, then it should be removed. Commented out code is even harder to maintain, so either include it (if it's needed) or remove it entirely. In case it's one of the situations when it's really needed to keep it this way: add some comment explaining why it is like this and when it can be removed or included.

Reply:
Done